### ISSUE TITLE
Issue with % class name

### ISSUE THREAD

**Issue by kaleidografik** [juhanakristian/tailwind-gradient-mask-image#1]
Hi @juhanakristian - I'm trying to use this plugin (super helpful), however am I right in thinking that using a % symbol as a class name isn't valid? I'm not able to get the plugin working in the browser, and in my compiled CSS code I'm being shown a red warning on on the class name. I wonder, as Tailwind core classes don't use this, if the use of the % symbol makes the CSS invalid?

Any help would be much appreciated.

**Comment by juhanakristian** [comment]
Hi @kaleidografik 👋 And thanks for posting this issue

Could you post a short code sample? Also which version of Tailwind and which browser are you using?

Does the playground work on your browser https://play.tailwindcss.com/ZAQrbYa3wH 

TBH I'm not completely sure if `%` is invalid on some browsers but it would be easy to remove it from the plugin it's causing problems.

**Comment by kaleidografik** [comment]
Sure. The playground does work, however I'm installing via npm and using the plugin version. It appears to be compiling correctly, however that compiled code does not translate to the browser at all (and is being flagged by my css linter as invalid with the % in the class name.

Compiled code:
```
.gradient-mask-b-0% {
  -webkit-mask-image: linear-gradient(to bottom, rgba(0, 0, 0, 1.0) 0%, transparent 100%);
  mask-image: linear-gradient(to bottom, rgba(0, 0, 0, 1.0) 0%, transparent 100%);
}
```

Using Tailwind 2.2.6 and using the JIT engine. Tested in Safari and Chrome.

Thanks.

**Comment by juhanakristian** [comment]
Thanks. I will probably drop the `%` symbol from the class name since it's not that important anyway.

**Comment by kaleidografik** [comment]
Brilliant, thank you. Could you let me know when the update is pushed?

I have tested and can confirm that without the %, the compiled code works perfectly!

**Comment by juhanakristian** [comment]
Yeah, sure. I will update this issue when the new version is available.

**Comment by juhanakristian** [comment]
This has now been fixed in version 1.0.0. These changes are breaking so I've updated the major version.

Thanks for reporting @kaleidografik 👍

**Comment by kaleidografik** [comment]
Thank you @juhanakristian - a great fix and appreciate the swift action so that we can use this plugin. Much appreciated!
